Assets/Scripts/Player/Movement/MouseLook.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MouseLook : MonoBehaviour
```

Comment:
```suggestion
public sealed class CameraController : MonoBehaviour
```
Sealed class en renaming.